dev-libs/nmeap/nmeap-0.3.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Extensible NMEA-0183 (GPS) data parser in standard C"
+HOMEPAGE="http://nmeap.sourceforge.net/"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc test-programs"
+
+DEPEND="doc? ( app-doc/doxygen )"
+
+src_prepare() {
+	default
+
+	# Repsect users CFLAGS and LDFLAGS for the static lib archive
+	sed -i -e 's/CFLAGS =/CFLAGS +=/' -e 's/-g -O0 -Werror -Wall/$(LDFLAGS)/' src/Makefile || die
+
+	# Silent output of Doxygen and update it, since it is quite old
+	if use doc; then
+		sed -i -e 's/QUIET.*/QUIET = YES/' Doxyfile || die
+		doxygen -u Doxyfile 2>/dev/null || die
+	fi
+
+	# Respect users CFLAGS and LDFLAGS for the test programs
+	# Only build those test programs, if a user wants it
+	if use test-programs; then
+		sed -i -e 's/-g -O0//' -e 's/-Wall -Werror/$(CFLAGS) $(LDFLAGS)/' tst/Makefile || die
+	else
+		sed -i -e '/TST/d' Makefile || die
+	fi
+}
+
+src_install() {
+	insinto /usr/include
+	doins inc/nmeap.h inc/nmeap_def.h
+
+	insinto /usr/$(get_libdir)
+	doins lib/libnmeap.a
```

Comment:
please point `dolib.a` without manual defenitions here

Reply:
Done. Also did ```doheader``` for above.